### DISCUSSION TITLE
What is the recommended method to implement a "contact" page?

### DISCUSSION THREAD

**Discussion by rcdailey** [adityatelange/hugo-PaperMod#503] [Q&A] [answered]
I am creating a blog using hugo + modpaper for use professionally. If potential employers would like to contact me, I'd like them to be able to do that from my blog.

What is the recommended method of doing this? My google search shows that Hugo doesn't have support for input forms. Should I just put an email somewhere like `blog@firstlast.com`? Should it be a separate page (with a link in the top right) or in the main page somewhere?

Thanks in advance for any ideas and examples!

**Comment by danielfdickinson** [comment, marked as answer]
It depends on what you are using for hosting. If Netlify, they offer a method of creating and using forms. If something without that option, then you need to look at something 'formspree' or other "contact form service" (web search; I use Netlify, so I have no means to guide you on other options).

I recommend against a public email address unless you enjoy spam.

**Reply by rcdailey** [reply]
I'm new to blogging so I appreciate the general advice. I am using Netlify. I'll search their docs for any help on setting it up. Thank you for your help!

**Reply by danielfdickinson** [reply]
Since you are using Netlify, you could use a shortcode like the one here: https://github.com/danielfdickinson/netlify-new-oldnew-mashup/blob/main/layouts/shortcodes/contact/basic-form.html combined with a content page such as: https://github.com/danielfdickinson/netlify-new-oldnew-mashup/blob/main/exampleSite/content/siteentry/contact.md
Of course, that is just one of many options, but I hope it helps!
